DXRando/DeusEx/Classes/TrashCanCommon.uc

```diff
@@ -0,0 +1,69 @@
+class TrashCanCommon extends DXRBase;
+
+static function DestroyTrashCan(Containers trashcan)
+{
+    local int i;
+	local Vector loc;
+	local TrashPaper trash;
+	local Rat vermin;
+    local Containers trashbag;
+    local float scale;
+
+	// trace down to see if we are sitting on the ground
+	loc = vect(0,0,0);
+	loc.Z -= trashcan.CollisionHeight + 8.0;
+	loc += trashcan.Location;
+
+	// only generate trash if we are on the ground
+	if (!trashcan.FastTrace(loc))
+	{
+		// maybe spawn some paper
```

Comment:
should paper be spawned if a trashbag is being spawned?

Reply:
It's junk that didn't fall into the trashbag in my imagination. Maybe it should be much less likely if a trashbag spawns?